sample/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,8 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
 		  package="com.lush.views.sample">
 
+	<uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Why do we need the internet permission?

Reply:
This is to test the webview with a url in sample app